src/app/(BottomTabNavigation)/Cases/styles.ts

```diff
@@ -7,4 +7,9 @@ export default StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
   },
+
+  scanner: {
```

Comment:
The `scanner` styles is located in `/src/app/(BottomTabNavigation)/Cases/QRCodeScanner/styles` so let's remove this